webview/src/experiments/components/table/body/RowContextMenu.tsx

```diff
@@ -324,7 +327,7 @@ const getContextMenuOptions = (
 
 export const RowContextMenu: React.FC<RowProp> = ({
   row: {
-    original: { status, starred, id, executor },
+    original: { branch, status, starred, id, executor },
```

Comment:
[I] When sending back a list of IDs (e.g. for removal) can we ensure that the list is unique, please?